src/requestCertificate.jsx

```diff
@@ -236,12 +230,19 @@ export class RequestCertificate extends React.Component {
         super(props);
         this.state = {
             showDialog: false,
+            hostname: "",
         };
 
         this.onOpen = this.onOpen.bind(this);
         this.onClose = this.onClose.bind(this);
     }
 
+    componentDidMount() {
+        cockpit.file("/etc/hostname", { superuser: "try" }).read()
+                .done((content, tag) => this.setState({ hostname: content.trim() })) // TODO: try required?
```

Comment:
What do you mean by this TODO?

Reply:
Dropped, I was wondering if this shouldn't use dbus to obtain the hostname wen for example `/etc/hostname` is empty